config/default.json

```diff
@@ -82,7 +82,7 @@
     "s2s": "http://rpe-service-auth-provider-prod.service.core-compute-prod.internal",
     "termsAndConditions": "http://xui-terms-and-conditions-aat.service.core-compute-aat.internal",
     "payment_return_url": "https://paymentoutcome-web.prod.platform.hmcts.net/",
```

Comment:
Are you sure about this change? If you doing this then you need to change Reference.ts accordingly

Reply:
Yes, this is what is actually within Reference - unfortunately due to people's merge conflicts I think this file keeps changing back - it should be judicialBookingApi in all instances